### ISSUE TITLE
Better websocket debugging

### ISSUE THREAD

**Issue by tresf** [qzind/qz-print#167]
@bberenz While troubleshooting #165, I'm finding the `qz.DEBUG = true;` to have limited benefit debugging websocket connection issues.

Furthermore, there seems to be no public method available for setting `trace` debug logs without editing `qz-tray.js`.

Perhaps... 

``` diff
try {
+   _qz.log.trace("WebSocket connection attempt: " + address);
    _qz.websocket.connection = new WebSocket(address);
}
```

As well as something like... 

``` js
qz.websocket.debug(true);  // set `DEBUG=true` to enable the `trace` logs
```

**Comment by akberenz** [comment]
There's some more debug statements added in df266a939fff6c76318de57b768e9f41f0f57e41. And a public method `qz.api.showDebug(<bool>)`.
This should cover more interactions, but I'm not sure if there's other spots we would want debug lines.

**Comment by tresf** [comment]
Looks good!  We can add more as needed.

**Edit:** Added to documentation https://qz.io/wiki/FAQ#debugging-websockets-in-20
**Edit:** Missing `@memberof qz.api` so not listing in API yet.

Closed via df266a9.

**Comment by tresf** [comment]
@bberenz I think we missed the API documentation here...
